keyboards/draytronics/elise/config.h

```diff
@@ -0,0 +1,19 @@
+/*Copyright 2021 Blake Drayson / Draytronics
+Contact info@draytronics.co.uk
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+//Fix to prevent key repeat on wake from sleep when using a KVM / non direct usb connection.
```

Comment:
This is something that really should be applied at a keymap/user level.

Reply:
This has been moved to the user level.